Pid.hpp

```diff
@@ -15,11 +17,20 @@ class Pid {
     /** clear any windup */
     void clearWindup();
 
+
+    //tuner stuff
+    void initialize_tuner();
+
+    void set_from_tuner();
+
+    void start_cycle();
```

Comment:
Alternatively, another way to use this would be to have the pid tuner take in the pid and then just run the autotuner with the autotuner functions.